Forwarded: https://github.com/mroonga/mroonga/issues/298
Subject: Mroonga plugin build in MariaDB is not reproducible (Closes: #976984)
 This did not make Mroonga reproducible, so we might as well remove this later on.

--- a/storage/mroonga/vendor/groonga/configure.ac
+++ b/storage/mroonga/vendor/groonga/configure.ac
@@ -1505,7 +1505,7 @@ GRN_DEFS="$GRN_DEFS -DGRN_DEFAULT_RELATI
 AC_SUBST(GRN_DEFS)
 CFLAGS="$CFLAGS $OPT_CFLAGS "
 LIBS="$LIBS $WINDOWS_LIBS"
-AC_DEFINE_UNQUOTED(CONFIGURE_OPTIONS, "$ac_configure_args", "specified configure options")
+AC_DEFINE_UNQUOTED(CONFIGURE_OPTIONS, "$(echo $ac_configure_args | sed -e "s@$(pwd)@<builddir>@g")", "specified configure options")
 
 # For groonga.org
 AC_ARG_WITH(groonga-org-path,
